app/models/notification.rb

```diff
@@ -4,11 +4,11 @@
 # This class' responsibility is, given a notification blueprint, to determine
 # who are the notification recipients
 class Notification < ApplicationRecord
-  AUDIENCE_USER     = 'user'
```

Comment:
Many strings in this class would benefit from `# frozen_string_literal: true`, I don't see any that should be mutated/reinstantiated.

Reply:
This "magic comment" is already in the file.